app/(app)/(home)/[category]/[subcategory]/page.tsx

```
@@ -1,24 +1,25 @@
 interface SubcateogryPageProps {
-  params: Promise<{
-    category: string;
-    subcategory: string;
-  }>;
+  params: Promise<{ category: string; subcategory: string; }>;
+  searchParams: Promise<SearchParams>;
 }
 
-export default async function SubcateogryPage({ params }: SubcateogryPageProps) {
+export default async function SubcateogryPage({ params, searchParams }: SubcateogryPageProps) {
   const { category, subcategory } = await params;
```

ESLint

[error] 'category' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)